test/Dapr.E2E.Test.App/Dapr.E2E.Test.App.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>

```

Comment:
Can this multitarget?

`<TargetFrameworks>netcoreapp3.1;net5</TargetFrameworks>